packages/api/app/middleware.js

```diff
@@ -0,0 +1,26 @@
+const { EcsStartTaskError } = require('@cumulus/errors');
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/api' });
+
+// Catch and send the error message down (instead of just 500: internal server error)
+// Need all 4 params, because that's how express knows this is the error handler
+// eslint-disable-next-line no-unused-vars
+const defaultErrorHandler = (error, req, res, next) => {
+  logger.error(error);
+  return res.boom.badRequest(error.message, error);
+};
+
+// eslint-disable-next-line no-unused-vars
+const asyncOperationEndpointErrorHandler = async (err, req, res, next) => {
+  logger.error('Failed to run ECS task', err);
+  if (err instanceof EcsStartTaskError) {
+    return res.boom.serverUnavailable(`Failed to run ECS task: ${err.message}`);
```

Comment:
I'm not sure we want to be exposing an implementation detail like "we're using ECS for async operations."

```suggestion
    return res.boom.serverUnavailable(`Failed to start async operation: ${err.message}`);
```

Reply:
These errors would be obscured the user anyway because boom hides the message for any 5xx errors, but I can still make the change.